schema.ts

```diff
@@ -568,6 +568,31 @@ export interface StringDateTimeSchema extends BaseStringSchema<ValueHintType.Dat
   timeFormat?: string;
 }
 
+/**
+ * Enabling options that can be used with a {@link ImageSchema}.
+ */
+ export enum ImageState {
```

Comment:
It shouldn't be the same enum for the two properties, they just happen to have the same values right now. Make it two enums, named Outline and RoundedCorners.